core/src/main/resources/jenkins/split-plugins.txt

```diff
@@ -18,4 +18,4 @@ bouncycastle-api 2.16 2.16.0
 # JENKINS-47393
 command-launcher 2.86 1.0
 # JENKINS-22367 TODO: Update after split
-jdk-tool 2.100 1.0-SNAPSHOT
\ No newline at end of file
+# jdk-tool 2.100 1.0-SNAPSHOT
```

Comment:
`LoadDetachedPluginsTest` checks that everything in `split-plugins.txt` gets installed when crossing the version boundaries, so it requires `jdk-tool` to be listed in `war/pom.xml` if the `jdk-tool` entry is uncommented here. That doesn't work now because the maven reactor builds `war` before `jdk-tool-plugin`, but will be fixable once `jdk-tool` has a release version that can be used in this PR.